content/docs/rfcs/31/README.md

```diff
@@ -0,0 +1,21 @@
+---
+slug: 31
+title: 31/WAKU2-TRANSPORT
+name: Waku v2 Transport Specification
+status: draft
+editor: Reeshav Khan <reeshav@status.im>
+contributors:
+---
+
+`31/WAKU2-TRANSPORT` describes the specification for Protocols supported by Waku v2. While Waku v2 strives to be transport agonistic, however any underlying transport layer MUST be IETF standardized transport protocol, this document underlines the baseline for interoperability between different Waku v2 clients.
```

Comment:
Why does any underlying transport MUST be IETF standardized transport protocol? Seems like an odd requirement to me. For example, where would this leave things such as meshnets/MANETs that haven't yet been standardized by IETF?